script_test.go

```diff
@@ -1017,14 +1017,6 @@ func TestFindFiles_RecursesIntoSubdirectories(t *testing.T) {
 	}
 }
 
-func TestFindFiles_InNonexistentPathReturnsError(t *testing.T) {
```

Comment:
If it's a key part of `FindFiles` behaviour that it _doesn't_ return an error for missing paths, shouldn't we have a test for that?

Reply:
Yeah great point, pushed up a test